dist/cookie/{{cookiecutter.project_slug}}/.crowdbotics.json

```diff
@@ -1,6 +1,6 @@
 {
   "scaffold": {
     "type": "react-native",
-    "version": "1.1.0"
+    "version": "1.2.0"
```

Comment:
Will this be released before @danielsousaio 's branch? Because he bumped the version to `2.0.0`. I think his should be merged first and maybe then this would be `2.1.0`

Reply:
This is not upgrading the RN scaffold hence we're still on the v1.x.x. I've bumped it up to 1.2.0 from 1.1.0. Then on Daniel's script, we can include 1.2.0 as an upgradable version.

Reply:
I meant its not updating react native versions. It is part of scaffold. We're only changing circleci and fastlane config.

Reply:
This will get merged and released tmrw

Reply:
@GaboGomezT me and Abdullah had a chat and this is okay to merge, I will rebase this onto the #875 PR and this will go out as part of the 2.0.0 version release too.